db/test/versions/0003_test.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert').strict;
+const helper = require('../helper');
+const testing = require('taskcluster-lib-testing');
+const {UNDEFINED_TABLE} = require('taskcluster-lib-postgres');
+
+suite(testing.suiteName(), function() {
+  helper.withDbForVersion();
+
+  test('azure_queue_messages table created', async function() {
```

Comment:
Should we also test that downgrading removes this table?

Reply:
Good point -- I wrote this before downgrading was supported!